examples/tls-authentication/nginx/README.md

```diff
@@ -0,0 +1,56 @@
+# TLS termination
+
+This example demonstrates how to enable the TLS Authentication through the nginx Ingress controller.
+
+## Prerequisites
+
+You need a valid CA File, composed of a group of valid enabled CAs. This MUST be in PEM Format.
```

Comment:
Please add somewhere a doc of how to set this up using openssl, maybe in the prerequisites section, and link to it from here? https://github.com/kubernetes/ingress/blob/master/examples/PREREQUISITES.md
```
## Generating a CA 
Some examples might require an explicty CA. You can act as your own CA by running...


## Generating Client certs
Some examples require client certs. You can generate your own, and sign them using the CA generated above, as follows...
```